tools/run-csx.cmd

```diff
@@ -26,10 +26,10 @@ if not exist %TOOLS_DIR%\.nuget md %TOOLS_DIR%\.nuget
 copy /Y %NUGET_CACHE_DIR%\NuGet.exe %TOOLS_DIR%\.nuget\NuGet.exe > nul
 
 rem restore packages for build script
-%TOOLS_DIR%\.nuget\NuGet.exe restore %TOOLS_DIR%\packages.config -PackagesDirectory %TOOLS_DIR%\packages -Verbosity quiet
+%TOOLS_DIR%\.nuget\NuGet.exe install %TOOLS_DIR%\packages.config -OutputDirectory %TOOLS_DIR%\packages -Verbosity quiet -ExcludeVersion

```

Comment:
`nuget install` on a packages.config basically does the same thing as `nuget restore`, but has the `-ExcludeVersion` switch